core/domain/topic_services.py

```diff
@@ -246,6 +284,36 @@ def save_new_topic(committer_id, topic):
         })])
 
 
+def update_subtopic_page(topic_id, subtopic_id, change_list):
+    """Applies a changelist and updates the subtopic page model entry in the
+    datastore.
+
+    Args:
+        topic_id: str. The topic id in which the subtopic is present.
+        subtopic_id: int. The id of the subtopic.
+        change_list: list(TopiChange). The list of changes that are to be
```

Comment:
Typeinfo is misspelled (but wrong anyway, it should be list(SubtopicPageChange)).

Reply:
Done.

Reply:
This still looks wrong?

Reply:
Actually, there were some places where I forgot to change. Removed them :)